easybuild/easyconfigs/a/Atomsk/Atomsk-b0.9.10-goolf-1.7.20.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'Atomsk'
+version = 'b0.9.10'
+
+homepage = 'http://atomsk.univ-lille1.fr/index.php'
+description = "Atomsk is an Open Source command-line program dedicated to the creation, manipulation, and conversion of atomic systems"
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['http://atomsk.univ-lille1.fr/code/']
+sources = ['%(namelower)s_%(version)s.tar.gz']
+
+start_dir = 'src'
+
+buildopts = " atomsk LAPACK=${LIBLAPACK}"
+
+files_to_copy = [(['atomsk'], 'bin'), '../*']
```

Comment:
What else are you copying with `../*`? Please include some of it in `sanity_check_paths`?